### PR TITLE
[CDF-594] - Making cdf responsible for calling cpf's init

### PR DESCRIPTION
**Pull request by jvelasques**

### CODE REVIEW

**Comment by jvelasques** [comment]
@pamval please review

File: cdf-pentaho-base/src/org/pentaho/cdf/CdfLifecycleListener.java [hunk 1]
```diff
@@ -26,6 +26,8 @@
   private static final Log logger = LogFactory.getLog( CdfLifecycleListener.class );
 
   public void init() throws PluginLifecycleException {
+    // cdf will be responsible for calling cpf's init
+    super.init();
     // Initialize plugin
```
Comment (pamval) [inline cdf-pentaho-base/src/org/pentaho/cdf/CdfLifecycleListener.java]: if you're just calling super, why do we need to define init here ?

**Comment by jvelasques** [comment]
@pamval changed accordingly, please review

**Comment by pamval** [comment]
this needs a backport to 5.x.

this needs to be tested in a windows VM.

**Pull request merged**